jep/0000/README.adoc

```diff
@@ -123,6 +114,10 @@ public class SSHPrivateKeyResource extends APIResource
 ```
 The `type` parameter is used for deserialization too.
 
+Also note the "version" attribute. It is added by the API to all messages (the real message payload goes into `data`).
+This version attribute can be used to handle backward incompatible changes in the API. It would be initially `version: 1`
+in all messages.
+
```

Comment:
It might be helpful to show an example of how use can leverage the resource version to maintain backward compatibility.  I've seen systems where versioning resources were touted as a magic bullet to avoid compatibility issues but in reality turned out to be impractical or never used.

Reply:
It's important to note that the `version` attribute is there to handle changes in the part of the message generated by the API itself (which is currently one attribute only, `data`), whatever goes inside the `data` attribute belongs to implementations/extensions of the API and they are not subject of this version attribute.